src/features/hotspots/root/HotspotsPicker.tsx

```diff
@@ -1,11 +1,13 @@
 import { getCurrentPositionAsync } from 'expo-location'
-import React, { memo, useCallback, useMemo } from 'react'
+import React, { memo, useCallback, useEffect, useMemo, useState } from 'react'
 import { useTranslation } from 'react-i18next'
 import { useSelector } from 'react-redux'
 import NewestHotspot from '@assets/images/newestHotspot.svg'
 import NearestHotspot from '@assets/images/nearestHotspot.svg'
```

Comment:
it looks like the "nearest" and "followed" sort options use the same icon, maybe we could use a globe or something for "nearest"?

![image](https://user-images.githubusercontent.com/792845/113445929-939a8480-93ab-11eb-8abb-e2489ab558e8.png)

can be added in a separate PR since might not have a design for it